fuel-tx/src/transaction/types/create.rs

```diff
@@ -208,8 +195,8 @@ impl Chargeable for Create {
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
There are some areas in the code below which are affected by this change:
![image](https://github.com/FuelLabs/fuel-vm/assets/794823/9202f1f0-a81d-4388-b894-6ae0656f7fda)